exercise-src/react-native/13-http-requests/01-problem/package.json

```diff
@@ -3,46 +3,47 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "typecheck": "tsc --noEmit",
+    "android": "react-native run-android",
+    "android:linkLocalhost": "adb reverse tcp:7070 tcp:7070",
+    "api": "npm run linkLocalhost:android && place-my-order-api --port 7070",
```

Comment:
Since we switched linkLocalhost:android => android:linkLocalhost, we will need to update the api script accordingly.

Reply:
good catch. I do think thing is a good change, tho

Reply:
Yes, thanks! Fixed now.